src/V3LinkDot.cpp

```diff
@@ -916,9 +916,21 @@ class LinkDotFindVisitor final : public VNVisitor {
             iterateChildren(nodep);
             nodep->user4(true);
         } else {  // !doit
-            // Will be optimized away later
-            // Can't remove now, as our backwards iterator will throw up
-            UINFO(5, "Module not under any CELL or top - dead module: " << nodep << endl);
+            if (nodep->name() == "_V_type_parameters") {
```

Comment:
Prefer use a new "/*verilator" attribute instead of hard coding a magic module name.